.ci/magic-modules/create-pr.sh

```diff
@@ -34,22 +34,32 @@ if [ "$BRANCH_NAME" = "$ORIGINAL_PR_BRANCH" ]; then
   # There is no existing PR - this is the first pass through the pipeline and
   # we will need to create a PR using 'hub'.
   if [ -n "$TERRAFORM_REPO_USER" ]; then
-    pushd build/terraform
-
-    git log -1 --pretty=%B > ./downstream_body
-    echo "" >> ./downstream_body
-    echo "<!-- This change is generated by MagicModules. -->" >> ./downstream_body
-    if [ -n "$ORIGINAL_PR_USER" ]; then
-      echo "/cc @$ORIGINAL_PR_USER" >> ./downstream_body
-    fi
-
-    git checkout -b "$BRANCH_NAME"
-    if TF_PR=$(hub pull-request -b "$TERRAFORM_REPO_USER/terraform-provider-google:master" -F ./downstream_body); then
-      DEPENDENCIES="${DEPENDENCIES}depends: $TF_PR ${NEWLINE}"
-    else
-      echo "Terraform - did not generate a PR."
-    fi
-    popd
+    for VERSION in '' 'beta'; do
```

Comment:
Do we someday plan on an tpg-alpha, or something like that, as well?  Basically, I think that _either_ we should have this for-each thing in `.ci/ci.yml.tmpl` _and_ in here, or else neither, and just duplicate the code below here.  I'm inclined towards the first option, but if you think the second is better, or if you think as-written this strikes the right balance, that also makes sense to me.